src/screens/Sidebar/PrivacySettings.js

```diff
@@ -15,9 +16,56 @@ class PrivacySettings extends Component {
     ),
   });
 
+  state = {
+    loading: true,
+    shareToGlobal: false,
+  };
+
+  componentDidMount() {
+    this.fetchPrivacySettings();
+  }
+
+  onSavePrivacySettings = async () => {
+    const { accessToken, currentUser } = this.props;
+    setToken(accessToken.token);
+    this.setState({ loading: true });
+    try {
+      await Kitsu.update('users', { id: currentUser.id, shareToGlobal: this.state.shareToGlobal });
+      this.setState({
+        shareToGlobal: this.state.shareToGlobal,
+        loading: false,
+      });
+    } catch (e) {
+      this.setState({
+        error: 'Failed to set privacy settings',
+        loading: false,
+      });
+    }
+  };
+
+  fetchPrivacySettings = async () => {
+    const token = this.props.accessToken;
+    setToken(token);
+    try {
+      const user = await Kitsu.findAll('users', {
+        fields: { users: 'shareToGlobal' },
+        filter: { self: true },
+      });
+      this.setState({
+        loading: false,
+        shareToGlobal: user[0].shareToGlobal,
+      });
+    } catch (e) {
+      this.setState({
+        error: 'Failed to fetch privacy settings',
+        loading: false,
+      });
+    }
+  };
+
```

Comment:
Should we keep this in redux states? I don't think it's being used anywhere else, if so, do tell and I move it